src/components/missions.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable consistent-return */
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Mission from './mission';
+import { loadMisionsThunk } from '../redux/missions/missionReducer';
+import './mission.css';
+
+function Missions() {
+  const state = useSelector((state) => state.missionReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (state.length === 0) {
+      dispatch(loadMisionsThunk());
+    }
+  }, []);
```

Comment:
- [ ] My comment on only fetching data on the first time a component is rendered also applies here.

Reply:
Thank you for this.
The issue just got resolved.

Reply:
Done ✔️